packages/commonwealth/server/routes/authCallback.ts

```diff
@@ -24,6 +24,8 @@ const authCallback = async (
   req: TypedRequestQuery<AuthCallbackReq>,
   res: TypedResponse<AuthCallbackResp>,
 ) => {
+  console.log('***********************authCallback', req.user);
```

Comment:
@jnaviask @kurtisassad I'll remove this log before merging... just trying to trace here...logging in and out with magic or Metamask is not reaching this point.

Reply:
@jnaviask after debugging with @kurtisassad, seems like this callback (including endpoint) has been deprecated and replaced by the Magic passport strategy. Can you confirm this before I try to remove it? 

If removed, the magic strategy above (`/server/passport/magic.ts`) is the only point where we create SsoToken records, and seems to confirm that issuer and address id fields are not optional.

Reply:
@Rotorsoft This route is used by discord bot for external linking, cc @timolegros -- can we remove this?

Reply:
For more context it was added in this PR:
https://github.com/hicommonwealth/commonwealth/pull/2099

Along with a call from the front end, but this call was later removed. But maybe it is used now for the discord bot.